src/app/tests/suites/certification/Test_TC_S_2_5.yaml

```diff
@@ -227,11 +277,12 @@ tests:
       disabled: true
 
     - label:
-          "Step 8b: Verify that the DUT sends a report data for
-          RemainingCapacity after the MinIntervalFloor time; store this value
-          into RemainingCapacity; verify RemainingCapacity equals
+          "Step 8b: Verify that the DUT sends a report data for FabricSceneInfo
+          after the MinIntervalFloor time; store the RemainingCapacity field
+          from this fabric’s entry reported in FabricSceneInfo into
+          RemainingCapacity; verify RemainingCapacity equals
           (MaxRemainingCapacity-2)."
-      PICS: S.S.A0007
+      PICS: S.S.A0002
```

Comment:
given that these are mandatory, I would suggest just removing them - they're more likely to cause bugs than help.